bin/stellar-core-backup

```diff
@@ -17,20 +18,25 @@ OptionParser.new do |opts|
     options[:list] = l
   end
   opts.on("-r", "--restore", "offline restore, uses latest backup") do |r|
-    options[:restore] = r
+    options[:type] = 'restore'
+  end
+  opts.on("-R", "--rm", "auto-clean buckets directory") do |r|
+    options[:clean] = 'R'
+  end
+  opts.on("--no-verify", "Don't PGP sign backed up shasum file or verify restore files integrity (default on)") do |r|
+    options.delete(:verify)
+  end
+  opts.on("-d", "--debug", "dump debug output") do |r|
```

Comment:
I can not find any reference to the $debug global, is this a vestige of your testing/debug code ?

Reply:
Yes, though I thought I had not actually pushed that up to the branch. It will be removed before the push to the Gems repo.